src/functions-and-arrays.js

```diff
@@ -78,7 +130,16 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(wordsCount, word9 ) {
+  if (wordsCount.length === 0 ) {
+    return 0 }
+     x =0
```

Comment:
when you want to declare a varible , need to use let/const word 
also it is better you choose a meaningful name for your variable ;) in this case, you can write counter instead of x 
let counter = 0